deployments/tests/test_dist.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+import pytest
+
+directories = ['dist/abis/goerli', 'dist/abis/mainnet', 'dist/artifacts/mainnet', 'dist/artifacts/goerli']
+
+
+def is_json_file(file):
+    return file.endswith('.json')
+
+
+def is_valid_json(file):
+    try:
+        with open(file) as f:
+            json.load(f)
+        return True
+    except json.JSONDecodeError:
+        return False
+
+
+@pytest.mark.parametrize("directory", directories)
+def test_directory_exists(directory):
+    assert os.path.exists(directory), f'{directory} does not exist'
+
+
+@pytest.mark.parametrize("directory", directories)
+def test_json_files(directory):
+    assert os.path.exists(directory), f'{directory} does not exist'
+
+    json_files = [f for f in os.listdir(directory) if is_json_file(f)]
+    assert len(json_files) > 0, f'No JSON files in {directory}'
+
+    for file in json_files:
+        assert is_valid_json(os.path.join(directory, file)), f'File {file} in {directory} is not valid JSON'
```

Comment:
I'm thinking... is there a way to test ("import") the newly generated npm package before publishing?
Because then we could actually have a proper test (JS test) that imports the package and does some basic sanity checks, e.g. for existence of an address of a certain contract etc.

Reply:
hm, good idea. Do I test for example just the RequestManager? Or do we test all currently available contracts?

Reply:
> hm, good idea. Do I test for example just the RequestManager? Or do we test all currently available contracts?

It depends on how deeply you want to test. Personally, I would at least make sure that all contracts have addresses and that there are at least `RequestManager` and `FillManager` there -- that would be sufficient as a smoke test, IMO.